ui/app/components/wizard/features-selection.js

```diff
@@ -6,18 +6,33 @@ import { FEATURE_MACHINE_TIME } from 'vault/helpers/wizard-constants';
 export default Component.extend({
   wizard: service(),
   version: service(),
+  permissions: service(),
 
   init() {
     this._super(...arguments);
     this.maybeHideFeatures();
   },
 
   maybeHideFeatures() {
+    let features = this.get('allFeatures');
+    features.forEach(feat => {
+      feat.disabled = this.doesNotHavePermission(feat.requiredPermissions);
+    });
+
     if (this.get('showReplication') === false) {
       let feature = this.get('allFeatures').findBy('key', 'replication');
       feature.show = false;
     }
   },
+
+  doesNotHavePermission(requiredPermissions) {
```

Comment:
Could we leave a comment about the expected shape of `requiredPermissions` here?